lib/action_kit.rb

```diff
@@ -0,0 +1,3 @@
+module ActionKit
+end
+
```

Comment:
The Rails community would describe service objects the orchestrate a user’s interactions within an application's domain. They'll unfailingly bring together multiple points within out app.

Here I've created a small wrapper for ActionKit's API. It requires no knowledge of the app it lives within (although the `Helper` module) does blur the lines a little. This module could be extracted and used by any other application. So code that isn't domain specific should typically live inside `/lib`.